protocols/ext-foreign-toplevel-list-v1.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is this protocol added when it's in w-p?

Reply:
good question, I may have thought it's part of wlr but idk

anyways, I've removed it